Extensions/timestamps.js

```diff
@@ -160,41 +160,21 @@ XKit.extensions.timestamps = new Object({
 			return;
 		}
 
-		var url = "https://www.tumblr.com/svc/indash_blog?limit=1&offset=0&should_bypass_safemode=true&should_bypass_tagfiltering=true" +
```

Comment:
this url is like if an eight foot tall willow cryptid designed a helvetica ampersand list shirt

Reply:
it really is